packages/dashmate/src/config/configJsonSchema.js

```diff
@@ -532,23 +544,111 @@ export default {
                         $ref: '#/definitions/tenderdashNodeAddress',
                       },
                     },
+                    flushThrottleTimeout: {
+                      $ref: '#/definitions/duration',
+                    },
+                    maxPacketMsgPayloadSize: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
+                    sendRate: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
+                    recvRate: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
                   },
-                  required: ['host', 'port', 'persistentPeers', 'seeds'],
+                  required: ['host', 'port', 'persistentPeers', 'seeds', 'flushThrottleTimeout', 'maxPacketMsgPayloadSize', 'sendRate', 'recvRate'],
```

Comment:
why are these required?

Reply:
All options in the config are required. So users can see all available options when we render the config